library/ui/background_selector/background_selector.js

```diff
@@ -37,16 +32,20 @@ lizMap.events.on({
         }
 
         function getNextBaselayerImage(){
-            var nextbl = $('#switcher-baselayer-select option:selected').next('option');
+            var nextbl = $('lizmap-base-layers select option:selected').next('option');
             if(nextbl.length == 0)
-                nextbl = $('#switcher-baselayer-select option:first');
+                nextbl = $('lizmap-base-layers select option:first');
             var nextbl_text = nextbl.text();
             var nextbl_val = nextbl.attr('value');
             var cur_image_url = $('#baselayer-image-selector').css('background-image');
             var cur_image_spl = cur_image_url.split('/');
-            cur_image_spl.pop();
+            var cur_image = cur_image_spl.pop();
             var nextbl_image = cur_image_spl.join('/') + '/' + nextbl_val + '.png';
-
+            var begin = window.location.protocol + "//" + window.location.host + "/";
```

Comment:
What's the aim of this part ? (using relative path i guess) ?

the full path works for me

Reply:
this part was removed by  https://github.com/3liz/lizmap-javascript-scripts/commit/0cb93915d923996e8486e919bf82459708694cd1